src/content/docs/agile-handbook/heroing/what-is-a-hero.mdx

```diff
@@ -81,39 +81,80 @@ Common questions and requests include:
 * **Questions about status of a pull request or issue**. Check in and see if you can figure out, or pull in the assignee for that pull request if the status isn't clear.
 * **Questions about things we don't own (blog, API Explorer, newrelic.com, etc)**. Help them out by directing them to the appropriate Slack channel. (For a list of properties and their owner, see _Who owns the other wesbites?_ in Google Docs.) If you can't figure out who owns it, try asking the writing team in Slack.
 
-## Second-shift hero support
+## Second-shift hero support [#emea_support]
 
 Our Europe writers cover the 2nd shift heroing during their regular working hours. Unlike the US-based heroes, our Barcelona writers hero for an entire two-week sprint. Second-shift heroes cover both GitHub and Slack, but we don't expect that second-shift heroes will field every single request that comes in during their working hours since they're also carrying standard sprint duties during their hero shift.
 
-## Route and triage user feedback
-
-The hero is responsible for routing user feedback to liasions. This sections covers the entire lifecycle of a user feedback ticket:
-
-1. The hero is responsible for routing incoming tickets to the proper liaison each day
-    - The Hero leaves a comment saying the ticket is being triaged
-2. Each writer should spend 10-15 minutes/day triaging their assigned tickets 
-    - Writer either rewrites or closes the ticket
-      - If the writer closes the ticket, they must leave a comment explaining why.
-    - Writer needs to add: 
-      - Acceptance criteria
-      - User story
-      - Action items
-      - Resources 
-    - While rewriting, focus on making the ticket as swarmable as possible
-    - Assign a priority [See priority info in our team's `DOC Jira and Github fields - Project 401 - 2022` doc]
-    - Add the following label: `doc_groomed`
-3. Triaged tickets enter sprints in one of two ways:
-    - **Scrum leaders**: Bring in tickets based on marked priority
-    - **Liaisons**: Bring in high value issues
-    - **Squads**: During backlog grooming the team can chat about bringing in tickets to fill out a sprint
-
 <Callout variant="important">
 For writers in Europe:
 
 - Route incoming issues to the liaison as you have time. As writers in Europe are "eternal heroes" it isn’t fair to have you do this every day.
 - You should still triage issues routed to you from your liaisonships once per day.
  </Callout>
 
+## Customer feedback sidekick [#cf_sidekick]
```

Comment:
I think this section could use a bit of discussion on three things:

1. Why customer feedback matters and why we dedicate time to it
2. Why the sidekick is the way we do it
3. How we undertake larger quality projects taht look at _all_ feedback for an area, including feedback closed. (Meaning even feedback that's insta-closed isn't wasted.)

Reply:
Valid!